test/performance-charts/vitest.config.ts

```diff
@@ -6,15 +6,20 @@ export default defineConfig({
   plugins: [codspeedPlugin(), react()],
   test: {
     setupFiles: ['./setup.ts'],
-    environment: 'jsdom',
-    // browser: {
-    //   enabled: true,
-    //   headless: true,
-    //   name: 'chromium',
-    //   provider: 'playwright',
-    //   providerOptions: {
-    //     timeout: 60000,
-    //   },
-    // },
+    ...(process.env.VITEST_ENV === 'browser'
```

Comment:
Had to add a `VITEST_ENV` because I didn't want to apply the `getBBox` polyfill in the browser environment since it's only absent in JSDOM.